src/containers/StructurePage/StructureContainer.tsx

```diff
@@ -199,10 +202,10 @@ const StructureContainer = ({
 
   return (
     <ErrorBoundary>
-      <Wrapper>
-        <GridContainer breakpoint={breakpoints.desktop}>
-          {messageBox && <Column>{messageBox}</Column>}
-          <Column colEnd={7}>
+      <PageContent variant="wide">
+        <GridWrapper>
+          {messageBox && <MessageBoxWrapper>{messageBox}</MessageBoxWrapper>}
```

Comment:
Kan vel droppe wrapperen og flytte den en opp, kan du ikke?

Reply:
Kan det, men isåfall må jeg legge på spacing over `MessageBox`, derfor jeg tenkte det var nice den var inne i wrapperen for å sikre at det er likt, men samme for meg egentlig!